scripts/NFTView.cdc

```diff
@@ -0,0 +1,26 @@
+import NonFungibleToken from "../contracts/standard/NonFungibleToken.cdc"
+import MetadataViews from "../contracts/standard/MetadataViews.cdc"
+import NFTRegistry from "../contracts/NFTRegistry.cdc"
+import FIND from "../contracts/FIND.cdc"
+
+//Fetch a single view from a nft on a given path
+pub fun main(user: String, aliasOrIdentifier:String, id: UInt64, identifier: String) : AnyStruct? {
```

Comment:
This script should be named consistent with the others, if we want to prefix with get then we should do it all the places.